### ISSUE TITLE
"Fields" query param is missing from pagination links

### ISSUE THREAD

**Issue by harmenjanssen** [laravel-json-api/laravel#175]
When paginating a JSON:API URL with a `fields` parameter, the pagination links inside `next` and `prev` are missing the `fields` parameter. 

I know the [JSON:API spec on pagination](https://jsonapi.org/format/#fetching-pagination) doesn't have anything to say on the subject of whether or not this is _correct_, but it sure is an inconvenience! It would be nice if the `next`/`prev` URLs can be used as-is in a loop, instead of the developer manually having to repeat the `fields` parameter.

**Comment by lindyhopchris** [comment]
You are right - the `fields` value should be in the pagination links as I agree it should be possible to just use those links to get another page without having to modify anything.

I'm surprised this isn't work, as I thought it was! So will need to investigate.

**Comment by lindyhopchris** [comment]
Confirmed this is a bug, been able to reproduce it in a test. Still not sure why it's happening, so will need to dig some more.

**Comment by lindyhopchris** [comment]
Fixed in `2.1.2`. To upgrade:

```bash
composer require laravel-json-api/laravel --no-update
composer up laravel-json-api/*
```

**Comment by harmenjanssen** [comment]
That is awesome! Thanks so much. 🙂
